### DISCUSSION TITLE
[FEATURE]: Trending category and tags

### DISCUSSION THREAD

**Discussion by amitkumar-248** [etsy/open-api#742] [Ideas]
### Is there an existing ticket for this?

- [X] I have searched the existing tickets

### Describe the solution you'd like

my existing clients are requesting for trending category and   trending tags , but in v3 API this option is not available 

### Describe any alternatives you've considered

https://www.etsy.com/developers/documentation/reference/listing#method_gettrendinglistings

**Comment by amitkumar-248** [comment]
@etsyachristensen any plan for this implementation, getting many requests for these points, it was very useful in V2 version.

**Comment by arturtr** [comment]
We also were going to use it. Probably these data may be somehow gotten in another way in API v3?
